metricbeat/module/rabbitmq/node/_meta/fields.yml

```diff
@@ -110,10 +110,6 @@
       type: long
       description: >
         Number of messages which have been written to the message store.
-    - name: name
```

Comment:
Shouldnt' this become an alias to `rabbitmq.node.name` as well?

Reply:
Fixed.